network/NetworkGen/NetworkGen.py

```diff
@@ -888,6 +919,32 @@ def calculate_bus_load(bus, output_dir):
     if usage > 1:
         raise Exception(f"CAN Bus '{bus.name.upper()}' has total bus usage of {usage}")
 
+
+def generate_manifest(bus: str, manifest: str, filters: list[str], ignore_nodes: list[str]):
+    nodes = {}
+
+    for node, node_def in can_nodes.items():
+        if node in ignore_nodes:
+            continue
+        for message, defs in { **node_def.messages, **node_def.received_msgs }.items():
+            for filter in filters:
+                alt = None
+                if '=' in filter:
+                    alt = filter.split('=')[1]
+                    filter = filter.split('=')[0]
+                if filter in message:
+                    node = node_def.alias
+                    if node not in nodes:
+                        nodes[node] = {}
+
+                    if nodes[node]:
```

Comment:
I think this is always true because of the check on line 937?

Reply:
lmao